azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -401,7 +401,7 @@ def _cancel_token_update_alarm(self):
         old_alarm = self._token_update_alarm
         self._token_update_alarm = None
         if old_alarm:
-            logger.debug("Cancelling SAS Token update alarm")
+            logger.debug("{}: Cancelling SAS Token update alarm".format(self.name))
```

Comment:
>logger.debug("{}: Cancelling SAS Token update alarm".format(self.name)) [](http://example.com/codeflow?start=12&length=71)

Just out of curiosity, did you add this for consistency or does self.name give you some extra info that was useful in debugging this?